build.gradle.kts

```diff
@@ -4,7 +4,7 @@ plugins {
     `java-library`
     `maven-publish`
     id("com.github.hierynomus.license") version "0.16.1"
-    id("com.github.johnrengelman.shadow") version "8.1.0"
+    id("com.github.johnrengelman.shadow") version "8.1.1"
```

Comment:
Should we consider switching to a fork? The original became EOL ([Related issue](https://github.com/johnrengelman/shadow/issues/908)).

I can't yet see a reason, but still wanted to bring it up. [Here](https://plugins.gradle.org/plugin/io.github.goooler.shadow) is a recommended fork

Reply:
Oh wow, I didn't even realize that! I'll take a look at the recommended fork you linked, thank you.